cartography/intel/aws/s3.py

```diff
@@ -17,14 +17,40 @@ def get_s3_bucket_list(boto3_session):
     return client.list_buckets()
 
 
+def get_s3_clients(boto3_session):
+    """
+    Generates S3 clients for all active regions
+
+    Arguments:
+        boto3_session {boto3.session} -- Current AWS Session
+
+    Returns:
+        dict -- dictionary of regional s3 clients
+    """
+    ec2 = boto3_session.client('ec2')
+    resp = ec2.describe_regions()['Regions']
+    active_regions = [
+        x['RegionName'] for x in resp
+        if x['OptInStatus'] in ['opt-in-not-required', 'opted-in']
```

Comment:
Ah I see, so this would only return regions that don't require being opted in?  I didn't know AWS had a concept like this 😅